driftbase/match_placements.py

```diff
@@ -208,6 +203,104 @@ def start_lobby_match_placement(player_id: int, queue: str, lobby_id: str) -> di
 
         return match_placement
 
+def start_match_placement(player_id: int, queue: str, map_name: str, max_players: int, identifier: typing.Optional[str] = None, custom_data: typing.Optional[str] = None) -> dict:
+    player_match_placement_key = _get_player_match_placement_key(player_id)
+
+    # Check existing placement
+    existing_placement_id = _check_existing_match_placement(player_id)
```

Comment:
Don't you want to do something with a non-None result here?

Reply:
No.
I only use it to verify the value is the same when I check it again later in case of any potential race conditions